build/src/src/watchers/nsupdate/index.ts

```diff
@@ -70,9 +70,11 @@ setIntervalDynamic(() => {
   nsupdateInterval //      60 min
 ]);
 
-// React immediatelly to new installs
 eventBus.packagesModified.on(({ ids, removed }) => {
```

Comment:
Event `packagesModified` is called when a container is recreated, on `updatePortMappings`, `updatePackageEnv`, `restartPackageVolumes`, `restartPackage`, `removePackage`, `installPackage`, `backupRestore`